packages/jest-snapshot/src/__tests__/snapshot_resolver.test.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
```

Comment:
genereal comment, `path.resolve` normalizes, so you don't need to comma separate the varargs

Reply:
👍 better now?

Reply:
…realized I only fixed it on the other files, also updated this one now.